### ISSUE TITLE
Jitsi config for local docker-compose deployment

### ISSUE THREAD

**Issue by mielune** [thecodingmachine/workadventure#683]
Hi all :)

I tried this realy fun tool today, and I planned to test the jitsi part, but it seems to not been configured.

Is it possible to run the stack with docker-compose with the rooms jitsi working ?

Kind regards,

**Comment by mielune** [comment]
I think it is not related to https://github.com/thecodingmachine/workadventure/issues/574
The Jitsi config needs documentation. We don't know if it can work with *Jitsi cloud service* or only with a dedicated on premise instance.

**Comment by loelkes** [comment]
I don't see any link to the issue you mentioned. Can you please elaborate _what exactly_ is not working for you?

All you have to do is let`JITSI_URL` in `.env` point to your Jitsi-Meet instance.

**Comment by mielune** [comment]
I setup the JITSI_URL env var in docker-compose.yaml here and it is not used : 
```
  front:
    image: thecodingmachine/nodejs:14
    environment:
      JITSI_URL: $JITSI_URL
```
I assume it is used to populate a `.env` during the startup of the front docker instance.
If I'm wrong, and `.env` is used to populate the `docker-compose.yaml`, I'm sorry, you can close this issue ^^

**Comment by moufmouf** [comment]
Hi guys,

Could you try with the new "production" docker-compose file that was merged in #698?

**Comment by gparant** [comment]
A contribution was merged, I close this issue.

Thanks for your feedback 💪 
If you have any other requirement, don't hesitate to create new issue 😍
